### ISSUE TITLE
Connect CI to coverage tool and add badge to readme

### ISSUE THREAD

**Issue by mashayev** [CityOfZion/neon-wallet#227]
Hi All,

I can tackle this, but I need more information. Where can I get the whole feature request?

**Comment by shawnmclean** [comment]
Hey @mashayev, we're using coveralls. We'd like to have our testing framework generate coverage reports during the CI test step. So it's just a matter of tweaking the `yarn test` command to output the coverage file in a way that coveralls will understand.

**Comment by shawnmclean** [comment]
You can also add the badge to readme, the md is:

`[![Coverage Status](https://coveralls.io/repos/github/CityOfZion/neon-wallet/badge.svg?branch=master)](https://coveralls.io/github/CityOfZion/neon-wallet?branch=master)`

**Comment by phetter** [comment]
I did a test, and accidentally committed it to another PR (which I reverted), that seemed to get this started. I simply added --coverage to the jest command in the tests script of the package.json like so:

  "scripts": {
    "postinstall": "electron-builder install-app-deps",
    "start": "electron .",
    "assets": "webpack",
    "assets-watch": "webpack --watch",
    **"test": "jest --coverage",**

When I ran yarn test it generated a coverage directory with the reports and listed a minimal report summary in the output of yarn test. If this is what you are looking for I can make the commit in a new PR once my other is closed.  Otherwise, I hope this helps to get you started on the right path.
